Model/Quote/GrandTotalDetailsPlugin.php

```diff
@@ -19,27 +19,57 @@ class GrandTotalDetailsPlugin extends \Magento\Tax\Model\Quote\GrandTotalDetails
      */
     protected $grandTotalRatesExtensionFactory;
 
+    /**
+     * @var \Magento\Tax\Api\Data\GrandTotalRatesInterfaceFactory
+     */
+    protected $ratesFactory;
+
+    /**
+     * @var \ReflectionClass
+     */
+    protected $reflectionClass;
```

Comment:
It looks like you were going to use this but then ended up needing not to? Remove it.